docs/units-and-conventions.rst

```diff
@@ -118,3 +118,23 @@ If we had a fully sampled grid of :math:`{\cal V}_{u,v}` values, then the operat
     I_{l,m} = U V (\Delta u)(\Delta v) \mathtt{iFFT}({\cal V}_{u,v})
 
 For more information on this procedure as implmented in MPoL, see the :class:`~mpol.gridding.Gridder` class and the source code of its :func:`~mpol.gridding.Gridder.get_dirty_image` method. When the grid of :math:`{\cal V}_{u,v}` values is not fully sampled (as in any real-world interferometric observation), there are many subtleties beyond this simple equation that warrant consideration when synthesizing an image via inverse Fourier transform. For more information, consult the seminal `Ph.D. thesis <http://www.aoc.nrao.edu/dissertations/dbriggs/>`_ of Daniel Briggs.
+
+
+"Packed" Image Cubes
+--------------------
+
+Internally, data is represented by Cubes, or arrays of three dimensions. In the case of MPoL, the dimensions of an MPoL cube represent Declination (DEC, axis one), Right Ascension (RA, axis two), and Velocity/Frequency (axis three). Initially, the Normal Cube, or Sky Cube, is the orientation of an image on the sky. Due to this, the RA increases to the *left*. It is desirable to flip the RA axis to obtain a traditional scaling with RA increasing to the *right*.
+
+We now have a flipped version of the Sky Cube (referred to as the Flip Cube). The next step is to "pack" the cube. In this application, this is carried out by applying ``torch.fft.fftshift()`` to the Flip Cube. This method acts only along the DEC and RA axes (axis one and axis two) leaving axis three untouched and creating a Packed Cube, or Image Cube. This method is represented internally as ``mpol.utils.sky_cube_to_packed_cube()``. 
+
+At this point, a two-dimensional Fast Fourier Transformation (described above) is applied to the Image Cube. Now the data has been transformed from a Sky Cube into what we refer to as a Visibility Cube with axes :math:`u,v`. The Visibility Cube is also a Packed Cube similar to the Image Cube, but on the other side of the FFT. By taking this and applying ``torch.fft.fftshift()`` along axes one and two we get a Ground Cube. This method is represented internally as ``mpol.utils.packed_cube_to_ground_cube()``. 
```

Comment:
I agree that I think it would be very helpful to have a graphical illustration of this, showing the array at each stage: sky, flipped, packed, with labeled coordinates. I would use the ALMA logo. You should be able to download it here: https://zenodo.org/record/4711811#.YKf3jJNKidY as the logo_cont.fits. 

Ulitmately, you'll want to include these as products built as part of the documentation Makefile and then included/referenced from this rst document. Just like the CSV file with baseline lengths.

Reply:
I downloaded the logo_cont.fits file, but the data is 2-dimensional, so I am not sure how to use this as a cube. I could use the .npz file instead if that works?